process/transaction/shardProcess.go

```diff
@@ -119,20 +141,111 @@ func (txProc *txProcessor) ProcessTransaction(tx *transaction.Transaction) error
 	return process.ErrWrongTransaction
 }
 
+func (txProc *txProcessor) createReceiptsWhenFail(tx *transaction.Transaction, acntSnd state.AccountHandler) error {
```

Comment:
this is more like "executingFailedTransaction" as it modifies the balance, increases the nonce, creating receipt

Reply:
changed the name.